client/src/i18n/locales/es/index.json

```diff
@@ -3,13 +3,60 @@
   "english": "English",
   "spanish": "Español",
   "signup": "Regístrate",
+  "or": "o",
   "login": "Inicia sesión",
+  "logout": "Cerrar sesión",
   "email": "Correo",
   "password": "Contraseña",
   "emailAddressRequired": "Correo electrónico es requerido",
   "passwordRequired": "Contraseña es requerido",
   "forgotPassword": "¿Olvidaste tu contraseña?",
   "resetPasswordText": "No te preocupes. Haz clic en el botón abajo y revisa tu correo para resetearla.",
   "resetPassword": "Resetea tu contraseña",
-  "pieforProvidersLogoAltText": "Logo de Pie for Providers"
+  "pieforProvidersLogoAltText": "Logo de Pie for Providers",
+  "agreeToTerms": "He leído y estoy de acuerdo con",
```

Comment:
Missing space at the end.

<img width="838" alt="image" src="https://user-images.githubusercontent.com/7039523/90841568-94057400-e322-11ea-8695-aefba551f95c.png">